CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 * [BUGFIX] Updated `golang.org/x/text` dependency to fix CVE-2022-32149. #3285
 * [BUGFIX] Query-frontend: properly close gRPC streams to the query-scheduler to stop memory and goroutines leak. #3302
 * [BUGFIX] Ruler: persist evaluation delay configured in the rulegroup. #3392
+* [ENHANCEMENT] Added Validation when S3 endpoint had a prefix of bucket name. #3414
```

Comment:
Sorry again, I mean that this should go after all the "[ENHANCEMENT]" elements.

Also could you please rehprase this as something like:
`S3 bucket configuration now validates that the endpoint does not have the bucket name prefix.`

(Why am I suggestion this change? This way the first words say what we're changing: S3 bucket configuration, so it's easier to visually scan for changes when something went wrong. Also lowercased the "validation word.)